src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Colors color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Creating a new instance of Random for each method call is inefficient. Instead, you should create a single Random instance as a class member and reuse it.